test/backends/fs_mac_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule FileSystem.Backends.FSMacTest do
+  use ExUnit.Case
+  import FileSystem.Backends.FSMac
+
```

Comment:
What do you think we should do for testing multiple backends in the future? 

My idea is to flag this test to only run when os_type is darwin, and then just make sure we run the full test suite on all the systems we support. Any other suggestions?